Code/Tools/ProjectManager/Source/ProjectsScreen.cpp

```diff
@@ -408,18 +409,61 @@ namespace O3DE::ProjectManager
             emit ChangeScreenRequest(ProjectManagerScreen::Projects);
         }
     }
+
+    static AZ::IO::FixedMaxPath GetEditorPathForProject(const AZ::IO::FixedMaxPath& projectPath)
```

Comment:
I'm OK with this location for now to fix this more pressing issue.  But I'd prefer in the long term we refactor `ProjectUtils::GetEditorDirectory` to take a ProjectPath param and possibly name the function `GetEditorExecutablePath` with it returning the full path to the editor binary.

There is another bit of code that needs the correct Editor path in ProjectButtonWidget.cpp - the code that generates a desktop shortcut to the editor.

Reply:
I can make that change before submitting this